### DISCUSSION TITLE
Farben in Terminal

### DISCUSSION THREAD

**Discussion by AnnaBlatt** [irgroup-classrooms/dis08-2021#15] [Q&A]
Hat jemand schon zufällig einen Weg gefunden, die verschiedenen Commands etwas farbenfroher zu gestalten bei Terminal? Die Grundeinstellungen bei Terminal finde ich nicht hilfreich (also über Einstellungen>Profile).

**Comment by MehmetCagferoglu** [comment]
Hello, du kannst einfach im Terminal Color und dann die Zahl der Farbe eingeben. Hier eine Anleitung " https://praxistipps.chip.de/cmd-farbe-aendern-so-gehts_47498?layout=amp ".
